RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHE_cfi.py

```diff
@@ -141,3 +141,7 @@
         allCellsPositionCalc = dict(logWeightDenominatorByDetector = {0 : dict(logWeightDenominator = _thresholdsHBphase1) } ),
     ),
 )
+
+# HCALonly WF
+particleFlowClusterHBHEOnly = particleFlowClusterHBHE.clone()
```

Comment:
better to modify parameters inside the clone statement (and avoiding repeating type spec):
```python
particleFlowClusterHBHEOnly = particleFlowClusterHBHE.clone(
    recHitsSource = "particleFlowRecHitHBHEOnly"
)
```
this should be applied to the other clone statements as well